README.md

```diff
@@ -548,7 +548,7 @@ pick 4b6e19a The second to last commit
 pick f4037ec The last commit
 ```
 
-This tells rebase that you want to edit your third last commit and keep the other two unaltered. Then you'll save (and close) the editor. Git will then start to rebase. It stops on the commit you want to alter, giving you the chance to edit that commit. Now you can apply the changes which you missed applying when you initially committed that commit. You do so by editing and staging them. Afterwards you'll run
```

Comment:
Afterwards is fine?

Reply:
Well that's silly.

Reply:
@RichardLitt  Do I keep it `Afterwards you'll run` and update the PR

Reply:
I'd prefer it. I would go back to Grammarly and tell them this is a useless change.